docs/next/guides/integrate-with-vue3/vue3-custom-editor-example.md

```diff
@@ -75,3 +75,38 @@ const app = createApp({
 app.mount('#example1');
 ```
 :::
+
+## Related articles
+
+#### Related guides
+
+- [Adding comments via the context menu](@/guides/cell-features/comments.md#adding-comments-via-the-context-menu)
+- [Clipboard: Context menu](@/guides/cell-features/clipboard.md#context-menu)
+- [Context menu](@/guides/accessories-and-menus/context-menu.md)
+- [Icon pack](@/guides/accessories-and-menus/icon-pack.md)
+
+#### Related blog articles
+
+- [Customize Handsontable context menu](https://handsontable.com/blog/customize-handsontable-context-menu)
```

Comment:
Check with Ola

Reply:
Ola says that for now it's OK (but they'll be double-checking blog posts soon).